ios/authentication.md

```diff
@@ -872,64 +873,136 @@ Now, your drop-in UI will show a Google sign in button which the users can use t
-- Click *User Pools* on the top menu to select a User Pool or create a new one.
-- Go to *Federation* > *Identity providers*
-- Select an *Identity provider* and enter required credentials for the identity provider. (e.g., App Id, App secret, Authorized scope)
-
-To learn [how to register with a Social IdP]({%if jekyll.environment == 'production'%}{{site.amplify.docs_baseurl}}{%endif%}/js/cognito-hosted-ui-federated-identity).
-{: .callout .callout--info}
-
-To learn [what's Authorized scope](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-social-idp.html#cognito-user-pools-social-idp-step-2)
-{: .callout .callout--info}
-
-Note: your user pool domain is something like: `domain_prefix.auth.us-east-1.amazoncognito.com`
-{: .callout .callout--info}
-
-- To retrieve user attributes from your identity provider, go to *Federation* > *Attribute mapping*. Here, you can map Federation Provider attributes to corresponding User pool attributes. More info about [Attribute Mapping](https://docs.aws.amazon.com/cognito/latest/developerguide/cognito-user-pools-specifying-attribute-mapping.html).
-
-If the attribute, for example *email*, is a required field in your Cognito User Pool settings, please make sure that you have selected *email* in your Authorized Scopes, and you have mapped it correctly to your User Pool attributes.
-{: .callout .callout-info}
-
-To setup App Client:
-- Go to [Amazon Cognito Console](https://aws.amazon.com/cognito/).
-- Click *User Pools* on the top menu to select a User Pool or create a new one.
-- Click *App integration*  and *App client settings* on the left menu.
-- Select *Enabled Identity Providers* and enter *Callback URL(s)* and *Sign out URL(s)* fields. 
+#### Automated Setup with CLI
+
+You would now need to configure your identity providers(Google, Facebook or Login with Amazon) which you would like to use.
+
+##### Setting Up OAuth With Facebook
+
+1. Create a [developer account with Facebook](https://developers.facebook.com/docs/facebook-login)
+2. [Sign In](https://developers.facebook.com/) with your Facebook credentials.
+3. From the *My Apps* menu, choose *Add New App*.
+![Image]({{common_media}}/cognitoHostedUI/facebook1.png)
+4. Give your Facebook app a name and choose *Create App ID*.
+![Image]({{common_media}}/cognitoHostedUI/facebook2.png)
+5. On the left navigation bar, choose *Settings* and then *Basic*.
+![Image]({{common_media}}/cognitoHostedUI/facebook3.png)
+6. Note the *App ID* and the *App Secret*. You will use them in the next section in the CLI flow.
+
+##### Setting up OAuth with Google
+
+1. Go to the [Google developer console](https://console.developers.google.com).
+2. On the left navigation bar, choose *Credentials*.
+![Image]({{common_media}}/cognitoHostedUI/google5.png)
+3. Create your OAuth2.0 credentials by choosing *OAuth client ID* from the *Create credentials* drop-down list.
+![Image]({{common_media}}/cognitoHostedUI/google6.png)
+4. Choose *Web application*.
```

Comment:
It asked me for configuring the consent screen after step 3 in Google console. Did we miss that by any chance ? We should capture that if it is the normal flow.

Reply:
I don't think so this is a part of the normal flow. I used this - https://aws-amplify.github.io/docs/js/cognito-hosted-ui-federated-identity as a reference for the google provider setup flow.

Reply:
I think I may know why it is different. The link the doc in the PR is https://console.developers.google.com whereas in the link you pointed out the link is https://developers.google.com/identity/sign-in/web/sign-in

Reply:
You need to have both of them. The second link comes in - in the section below during the final setup of OAuth with the hosted UI link which is an output of the `amplify push` command.